lib/_tls_wrap.js

```diff
@@ -494,9 +494,7 @@ function TLSSocket(socket, opts) {
     handle: this._wrapHandle(wrap),
     allowHalfOpen: socket ? socket.allowHalfOpen : tlsOptions.allowHalfOpen,
     pauseOnCreate: tlsOptions.pauseOnConnect,
-    // The readable flag is only needed if pauseOnCreate will be handled.
-    readable: tlsOptions.pauseOnConnect,
-    writable: false
+    manualStart: true
```

Comment:
What we actually want here is `manualStart` since we in the lines below actually do manually start it.

Reply:
`process.nextTick(initRead, this, socket);`